Modules/vc-module-catalog/VirtoCommerce.CatalogModule.Data/ExportImport2/ProductExportPagedDataSource.cs

```diff
@@ -65,6 +64,13 @@ protected virtual IEnumerable<IExportable> ToExportable(IEnumerable<ICloneable>
             var models = objects.Cast<CatalogProduct>();
             var viewableMap = models.ToDictionary(x => x, x => AbstractTypeFactory<ExportableProduct>.TryCreateInstance().FromModel(x));
 
+            foreach (var kvp in viewableMap)
```

Comment:
Seems we do not need dictionary here, it was used when we needed <entity, exportable> map to fill additional properties.
You could just get the result using simple select with property filling.